js/utils.js

```diff
@@ -1596,6 +1596,26 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
+  function toFormattedString(options) {
+    if (!options) {
+      return '';
+    }
+
+    if (Array.isArray(options)) {
+      options = options.filter(function(item) {
+        return !!item.trim();
+      });
+
+      options = options.join(', ');
+    }
+
+    if (typeof options === 'object') {
+      return JSON.stringify(options);
+    }
+
+    return options;
```

Comment:
In this case, a number will be returned as number. A number should be returned as the number in string format.